src/ast/mod.rs

```diff
@@ -7641,13 +7641,32 @@ impl fmt::Display for DuplicateTreatment {
     }
 }
 
+#[derive(Debug, Copy, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
+pub enum AnalyzeFormatKind {
+    Keyword(AnalyzeFormat),
+    Assignment(AnalyzeFormat),
+}
```

Comment:
Could we add some description ideally with example syntax to show the difference between each variant? Thinking that might not be obvious otherwise to someone looking at the API